src/sipsess/listen.c

```diff
@@ -245,6 +245,34 @@ static void invite_handler(struct sipsess_sock *sock,
 }
 
 
+/* Handle NOTIFY message
+ * Return false if message could not be handled, it will then fall through to
+ * out-of-dialog message handling.
+ */
+static bool notify_handler(struct sipsess_sock *sock, const struct sip_msg *msg)
+{
+	struct sip *sip = sock->sip;
+	struct sipsess *sess;
+
+	sess = sipsess_find(sock, msg);
+	if (!sess || sess->terminated) {
+		(void)sip_reply(sip, msg, 481, "Call Does Not Exist");
+		return true;
+	}
```

Comment:
perhaps return false here, so that the NOTIFY message can be handled
by the next listener ?

Reply:
done